pandas/core/frame.py

```diff
@@ -463,7 +463,8 @@ class DataFrame(NDFrame, OpsMixin):
     ----------
     data : ndarray (structured or homogeneous), Iterable, dict, or DataFrame
         Dict can contain Series, arrays, constants, dataclass or list-like objects. If
-        data is a dict, column order follows insertion-order.
+        data is a dict, column order follows insertion-order. If a dict contains Series
+        which has an index defined, it is aligned by its index.
```

Comment:
which have an index defined instead of has? Series references more than one like above?

Reply:
Thank you for review!
Series are more than one, so I modify it.